app/services/finance/npq/participant_aggregator.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    class ParticipantAggregator < Finance::ParticipantAggregator
+      def self.aggregation_types
+        {
+          started: {
+            all: :unique_for_lead_provider_and_course_identifier,
+            eligible_or_payable: :unique_for_lead_provider_and_course_identifier,
+            # not_paid: :submitted_for_lead_provider_and_course,
```

Comment:
✂️

Reply:
yea, i want to leave this to fix another time. it no longer makes sense in my opinion but would also need removing from the view